### PR TITLE
SteelSeries.SteelSeriesEngine version 3.22.1

### PR DESCRIPTION
**Pull request by spitfire**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/denelon/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/40868)

### CODE REVIEW

File: manifests/s/SteelSeries/SteelSeriesEngine/3.22.1/SteelSeries.SteelSeriesEngine.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SteelSeries.SteelSeriesEngine
+PackageVersion: 3.22.1
+PackageLocale: en-US
+Publisher: SteelSeries
+PackageName: SteelSeriesEngine
```
Comment (OfficialEsco) [inline manifests/s/SteelSeries/SteelSeriesEngine/3.22.1/SteelSeries.SteelSeriesEngine.locale.en-US.yaml:8]:
```suggestion
Publisher: SteelSeries ApS
PackageName: SteelSeries Engine
```

File: manifests/s/SteelSeries/SteelSeriesEngine/3.22.1/SteelSeries.SteelSeriesEngine.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+PackageIdentifier: SteelSeries.SteelSeriesEngine
+PackageVersion: 3.22.1
+PackageLocale: en-US
+Publisher: SteelSeries
+PackageName: SteelSeriesEngine
+PackageUrl: https://steelseries.com/engine
+License: Copyright (c) SteelSeries ApS
+ShortDescription: SteelSeries.SteelSeriesEngine
+Moniker: SteelSeriesEngine
```
Comment (OfficialEsco) [inline manifests/s/SteelSeries/SteelSeriesEngine/3.22.1/SteelSeries.SteelSeriesEngine.locale.en-US.yaml:12]:
```suggestion
License: Proprietary
Copyright: Copyright (c) SteelSeries ApS
ShortDescription: SteelSeries Engine is a configuration program for all compatible SteelSeries gear, allowing the user to adjust a wide range of customization options for their SteelSeries product(s).
Moniker: steelseries-engine
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]
Huh, this software have been forgotten..

**Comment by spitfire** [comment]
> Huh, this software have been forgotten..

I don't think this will get any more updates, since they moved on to a bigger suite of tools containing this. Not everyone likes to install whole bunch of stuff though.

**Review by OfficialEsco — APPROVED** [review]
Well, atleast now it can be exported and imported to the people who use the software :)

**Pull request merged**